foundations/javascript_basics/installing_nodejs.md

```diff
@@ -140,5 +140,8 @@ To run the Node console, open up your terminal and type `node`. Type `.exit` to
 ### Additional Resources
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+- [Installing Nodejs](https://nodejs.dev/en/learn/how-to-install-nodejs/)
```

Comment:
> I added some resources to the curriculum , under Nodejs 

Thank you for contributing @CodeLikeAGirl29, but can you please explain the "why" when adding new resources.

What value will these add to the lesson? what will learners get out of these new resources that they can't get from the lesson content already?

Some of our learners go through every single resource linked in our lessons. This could add many hours to their TOP journey, especially that last link. Theres needs to be a very good reason for adding these.